src/kimmdy/runmanager.py

```diff
@@ -41,8 +41,9 @@
 # file types of which there will be multiple files per type
 AMBIGUOUS_SUFFS = ["dat", "xvg", "log", "itp", "mdp"]
 # file strings which to ignore
-IGNORE_SUBSTR = ["_prev.cpt"]
+IGNORE_SUBSTR = ["_prev.cpt", ".start", ".done", ".failed", ".tail", "_mod.top"]
```

Comment:
While we are at it, can we make the names for theses files maybe include kimmdy somehwere in there name such that we always know which files are created by us, maybe kimmdy_... or kmdy_ for temporary, small things? 
And also make every filename that is used in more than one place a constant such that we can always ensure consistency (This avoids e.g. writing a `.done` file in one place and checking for a `.finished` file in another, which can be a pain to debug if those are independant strings).

Reply:
.kimmdy_done!